pyanaconda/ui/gui/spokes/network.py

```diff
@@ -747,10 +750,36 @@ def add_device_to_list(self, device):
         else:
             self.add_dev_cfg(DeviceConfiguration(device=device, con=con))
 
-        device.connect("notify::ip4-config", self.on_device_config_changed)
-        device.connect("notify::ip6-config", self.on_device_config_changed)
+        device.connect("notify::ip4-config", self.on_ip_obj_changed, IPV4_CONFIG)
+        device.connect("notify::ip6-config", self.on_ip_obj_changed, IPV6_CONFIG)
         device.connect("state-changed", self.on_device_state_changed)
 
+    def on_ip_obj_changed(self, device, *args):
+        """Callback when ipX-config objects will be changed.
+
+        Register callback on properties (IP address, gateway...) of these ipX-config
+        objects when they are created.
+        """
+        log.debug("%s object changed", args[1])
```

Comment:
anaconda.log is already rather cluttered with networking log messages, to mitigate this a bit I'd add "network:" prefix like in most of messages from network spoke